### ISSUE TITLE
API - POST playlists return error 500

### ISSUE THREAD

**Issue by stephane-r** [omarroth/invidious#1223]
Hello community,

I try using the very great invidious API. For know, i test all _playlists_ route. I've generated token with `scope: [*]` (^^) and i try to create new playlist from API : 

`POST /api/v1/auth/playlists` return me an error 500 with `Error: Failure when receiving data from the peer`. But, the playlist are been created (i see on website and `GET /playlists` return the good data).

Anyone know if this error is "logic", because i need other parameters, or maybe invidio.us problem ?

Thank you guys !

**Comment by cloudrac3r** [comment]
Scope needs to be `:*`, not `*`.

Are you using curl? Try adding `--http1.1`.

If you're using curl, and neither of these help, try adding `--trace -` and seeing what it produces.

**Comment by stephane-r** [comment]
Hi @cloudrac3r 
Yes sorry, the scope is `:*`, not `*`. For the request, i'm using HTTP call on mobile and web platform with `fetch`. For try my API Call, i use (Insomia)[https://insomnia.rest/] tool too.

I've trying with CURL, and the API has been return an good response with the JSON of the playlist created.

I'm not really understand why it work with CURL, and not with fetch : i've exactly the same body, headers, url and token :thinking:

**Comment by cloudrac3r** [comment]
Trying to use fetch on a webpage? You may be hitting restrictions imposed by the [same-origin policy.](https://web.dev/same-origin-policy/)

**Comment by stephane-r** [comment]
Yes, i've this error using fetch on webpage. Same on test soft API like Insomia.
